index.html

```diff
@@ -160,31 +149,19 @@ <h3 style="display: inline">LinkedIn</h3></a
       <div class="toggle">
         <img
           src="./assets/menu_30dp_1791C8_FILL0_wght400_GRAD0_opsz24.svg"
-          alt="menu"
-        />
+          alt="menu" />
       </div>
     </nav>
 
     <div style="margin-top: 10rem" class="ui container">
       <div class="ui segment">
         <!-- h1 used for SEO -->
-        <h1 class="h2-sized-header">About Me</h1>
-        <p>
-          Hi, I'm Michał Terczyński, a Frontend Developer with a strong focus on
-          game development and interactive applications. With experience at
-          companies like NetEnt, Yggdrasil Gaming, and Software Mind, I've
-          developed expertise in creating engaging slot games and enterprise
-          applications using TypeScript, React, Angular, and PixiJS.
-        </p>
-        <p>
-          My technical skills include modern JavaScript frameworks, game
-          development with PixiJS, and building scalable web applications. I'm
-          passionate about creating high-quality, performant code and enjoy
-          working on both commercial projects and personal game development
-          endeavors.
-        </p>
+        <h1 class="h2-sized-header" id="aboutTitle"></h1>
+        <p id="aboutIntro"></p>
+        <p id="aboutDetails"></p>
       </div>
-
+      <!-- loader -->
+      <div id="loader" class=""><div class="spinner"></div></div>
```

Comment:
Language translations are defined in the code, they will load synchronously, therefore I think we could remove the loader entirely